packages/compass-components/src/components/keyline-card.tsx

```diff
@@ -17,23 +20,74 @@ const keylineDarkThemeStyles = css({
   borderColor: palette.gray.dark2,
 });
 
+const lightHoverBoxShadow = `0 4px 20px -4px ${transparentize(
+  0.8,
+  palette.black
+)}`;
+
+const darkHoverBoxShadow = `0 4px 20px -4px ${transparentize(
+  0.3,
+  palette.black
+)}`;
```

Comment:
Looking closely this hover seems to go out of the boundaries and gets a cut off look:
<img width="305" alt="Screenshot 2023-10-19 at 11 28 16 AM" src="https://github.com/mongodb-js/compass/assets/1791149/c57d8aea-fda9-4869-9aaa-674820a6b6bb">
I see that this is the same as the [clickable card styles](https://github.com/mongodb/leafygreen-ui/blob/cf06988dcef8e2dac83ebc409f3b2e9a905fe9e8/packages/card/src/Card/styles.ts#L22), so I'm assuming we want to keep them.
Maybe we can make the container padding and margins set in the way where the items are not cut off?
Fine as is too, this is a design nit.

Reply:
probably done .. i moved the paddings around